tests/tools/test_basic.py

```diff
@@ -26,7 +26,7 @@ def runTest(self):
 class lookup_binary_in_path(libkcov.TestCase):
     @unittest.expectedFailure
     def runTest(self):
-        os.environ["PATH"] += self.sources + "/tests/python"
+        os.environ["PATH"] += os.pathsep + self.sources + "/tests/python"
```

Comment:
This was the patch needed to fix things up
```
 /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:./tests/python
```
without the separator nothing works correctly, and rm, mv and friends can not be found

Reply:
Wow, good catch!

I wonder why it was like that? I guess maybe self.sources was an absolute path in some cases, and a relative one in another. Anyhow, good that it's fixed!

Reply:
Best guess the last path in the list was not that important, but not on Alpine haha
Always add the separator 😄